src/Aspire.Hosting/Utils/PasswordGenerator.cs

```diff
@@ -49,4 +50,20 @@ public static string GeneratePassword(int lowerCase, int upperCase, int digit, i
 
         return new string(chars);
     }
+
+    /// <summary>
+    /// Creates a random string of upper and lower case letters.
+    /// </summary>
+    public static string GenerateRandomLettersValue(int minLength)
+    {
+        ArgumentOutOfRangeException.ThrowIfNegative(minLength);
+        ArgumentOutOfRangeException.ThrowIfGreaterThan(minLength, 128);
```

Comment:
nit, the check in the method above line 41 should be `ArgumentOutOfRangeException.ThrowIfGreaterThan(length, 128);` not just an assert

Reply:
The method above doesn't ever recent public input, which is why it uses an assert.

Reply:
Then why throw for negative inputs? But no biggie